docs-md/concepts/materialized-views.md

```diff
@@ -1,16 +1,53 @@
 ---
 layout: page
-keywords: ksqldb, query, select
+title: Materialized Views
+tagline: Aggregation over streams in ksqlDB
+description: Learn about aggregation and materialized views in ksqlDB.  
+keywords: ksqldb, query, select, aggregate, materialized view
 ---
 
-One of the main purposes of tables in any database is to enable efficient queries over the data. We’ve seen how ksqlDB stores events immutability in Kafka with a simple key/value model. How can queries be made efficient under this model? The answer is by leveraging materialized views. How this works requires a bit of an explanation.
+In any database, one of the main purposes of a table is to enable efficient
+queries over the data. ksqlDB stores events immutably in {{ site.aktm }} by
+using a simple key/value model. But how can queries be made efficient under
+this model? The answer is by leveraging materialized views.
 
-Streams and tables share an intimate relationship. A stream is a sequence of events from which a table can be derived. For example, a sequence of credit scores for a loan applicant can change over time. The sequence of credit scores is a stream. But that stream be interpreted as a table to describe the applicant’s current credit score. On the flip side, the table that represents current credit stores is really two things: the current credit scores, and also the sequence of changes to the credit scores for each applicant. This is a profound realization, and much has been written on this stream/table duality.  Many databases throw away the series of changes to a table and only keep the current state around, thus breaking the natural duality.
+Streams and tables are closely related. A stream is a sequence of events that
+you can derive a table from. For example, a sequence of credit scores for a
+loan applicant can change over time. The sequence of credit scores is a stream.
+But this stream be interpreted as a table to describe the applicant's current
+credit score.
 
-For our purposes, the important thing that enables efficient queries is access to the current state of applied changes. This is more commonly known as a materialized view. 
+Conversely, the table that represents current credit stores is really two
+things: the current credit scores, and also the sequence of changes to the
+credit scores for each applicant. This is a profound realization, and much has
+been written on this stream/table duality.  Many databases throw away the

```

Comment:
link to our blogs or the paper?

Reply:
Totally. @JimGalasyn If there's not a direct link, this is the one we want: https://www.confluent.io/resources/streams-tables-two-sides-same-coin?utm_source=blog&utm_medium=stream_processing&utm_campaign=two_sides_same_coin